cmd/cli/kubectl-kyverno/utils/store/store.go

```diff
@@ -64,6 +73,7 @@ type Policy struct {
 }
 
 type Rule struct {
-	Name   string            `json:"name"`
-	Values map[string]string `json:"values"`
+	Name          string              `json:"name"`
+	Values        map[string]string   `json:"values"`
+	ForeachValues map[string][]string `json:"foreachValues"`
```

Comment:
Duplicate definition, can we reuse the one in cmd/cli/kubectl-kyverno/utils/common/common.go?

Reply:
@realshuting they are not duplicated. Store values we are using in get get variables from files. 
also in pkg/engine mutation_test.go and validation_test.go